lib/widget/filters/remove_button.rb

```diff
@@ -24,8 +24,11 @@ class Widget::Filters::RemoveButton < Widget::Filters::Base
   def render
     hidden_field = tag :input, id: "rm_#{filter_class.underscore_name}",
                                name: 'fields[]', type: 'hidden', value: ''
-    button = icon_wrapper('icon-context icon-button icon-delete2 filter_rem', l(:description_remove_filter))
+    button = content_tag(:a, href: "#") do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.